src/main/java/seedu/duke/ui/Colors.java

```diff
@@ -0,0 +1,16 @@
+package seedu.duke.ui;
+
+/**
+ * Color class contains ANSI color codes for text color in the console.
+ */
+public class Colors {
```

Comment:
Good that you created a class to handle ANSI color codes! It will help make the codes relating to ui neater.